src/compiler/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.16)
```

Comment:
Do we really need to raise it here as well? The clang compiler plugin does not use OpenMP and is compiled with a regular, pure C++ compiler.

Reply:
Will do, no prob.